apps/studio/components/interfaces/Integrations/Vault/Secrets/SecretsManagement.tsx

```
@@ -52,32 +45,17 @@ export const SecretsManagement = () => {
 
   const {
     data,
+    error,
+    isError,
     isPending: isLoading,
     isRefetching,
     refetch,
-    error,
-    isError,
   } = useVaultSecretsQuery({
     projectRef: project?.ref!,
```

Biome (2.3.14)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)